intelligence/ts/src/typing.ts

```diff
@@ -260,6 +260,38 @@ export interface Progress {
   description?: string;
 }
 
+interface JSONSchema {
```

Comment:
For consistency with our naming scheme:
```suggestion
interface JsonSchema {
```